libraries/datadog_push.rb

```diff
@@ -0,0 +1,8 @@
+require 'datadog/statsd'
+
+def push_to_datadog(event)
+  if event && event[:name] && event[:text]
```

Comment:
Would be clearer if you move this to the provider itself and use chef's built-in `if{}` construct instead of this `if`.